lib/convict.js

```diff
@@ -140,12 +141,13 @@ function normalizeSchema (name, node, props, fullName, env, argv) {
     if (BUILT_INS.indexOf(format) >= 0) {
       // if the format property is a built-in JavaScript constructor,
       // assert that the value is of that type
+      var Format = typeof format == 'string' ? eval(format) : format;
```

Comment:
Would it suffice to use a simple global name->value map for this, like the following?

```
BUILT_INS_BY_NAME = {
  'Object': Object,
  'Array': Array,
  'String': String,
 ...
}

var Format = typeof format === 'string' ? BUILTINS_BY_NAME[format] : format
```

It's more verbose but avoids the need to `eval` anything.